src/icons/general/IconSecurityClassification.js

```diff
@@ -31,11 +31,11 @@ const IconSecurityClassification = ({
                 d="M17,2 L5,8 L5,15 C5,21.4214876 10.6933333,29.5421488 17,31 C23.3066667,29.5421488 29,21.4214876 29,15 L29,8 L17,2 Z"
                 stroke={color}
                 strokeWidth={strokeWidth}
-                fill="none"
+                fill={className === 'reverse' ? color : 'none'}
```

Comment:
Hm, I'm not sure we should be driving logic based on the class name. @booc0mtaco, thoughts?

Reply:
I don't think we want to do this. 

- `reverse` isn't a class name in this case, correct? so the prop isn't the value
- if `reverse` **is** a classname, it should be SUIT-able, correct?
- as a class name, we could specify the color to use via SASS

Reply:
Thanks! I reverted the update and have the inverted icon as a separated component.